axelar-chains-config/info/mainnet.json

```diff
@@ -1367,7 +1370,10 @@
         "gasPrice": 10000000000
       },
       "finality": "finalized",
-      "approxFinalityWaitTime": 25
+      "approxFinalityWaitTime": 25,
+      "gasOptions": {
+        "gasEstimationType": 1
```

Comment:
same as above. set the corresponding L1 name as well

Reply:
Added canonical chain names to the `onchainGasEstimate`